src/main/java/com/ticketbooking/repo/SeatRepo.java

```diff
@@ -0,0 +1,13 @@
+package com.ticketbooking.repo;
+
+import com.ticketbooking.model.Seat;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface SeatRepo extends JpaRepository<Seat, Long> {
+
+    List<Seat> findAllByRoomId(Long roomId);
```

Comment:
co się stanie jak podam `null`?

Reply:
Taka sytuacja jest nie jest możliwa bo w klasie Seat na polu room jest dodany warunek @JoinColumn(name = "room_id", nullable = false). Ale gdyby go nie było to zostaną zwrócone wiersze które spełniają taki warunek;